src/shared/components/QuickLaneModal/QuickLaneModal.tsx

```diff
@@ -51,7 +53,7 @@ const defaultQuickLaneState: QuickLaneUrlObject = {
 // Helpers
 
 const buildQuickLaneHref = (id: string): string => {
-	return `https://example.com/url/structure/${id}`;
+	return `${APP_PATH.QUICK_LANE.route.replace(':id', id)}`;
```

Comment:
react-router heeft hiervoor een helper: https://v5.reactrouter.com/core/api/generatePath
Deze gaat wel een error throwen als de params niet matchen met het path dus dat moet je dan nog wel opvangen.

Reply:
Fixed in https://github.com/viaacode/avo2-client/pull/1270/commits/5cb1d4ed6c3ef1a70c8535bf9a819eb8fc709428